5_data_munge/src/munge_wqp_files.R

```diff
@@ -93,19 +93,123 @@ munge_wqp_withoutdepths <- function(in_ind, min_value, max_value, max_daily_rang
                                        f_to_c(ResultMeasureValue), ResultMeasureValue)) %>%
     mutate(`ResultMeasure/MeasureUnitCode` = 'deg C')
 
-  dat_daily <- group_by(dat_reduced, MonitoringLocationIdentifier, ActivityStartDate) %>%
-    summarize(temperature_mean_daily = mean(ResultMeasureValue),
-              temperature_min_daily = min(ResultMeasureValue),
-              temperature_max_daily = max(ResultMeasureValue),
-              n_obs = n(),
+  dat_reduced_statcode <- ungroup(dat_reduced) %>%
+    filter(!is.na(StatisticalBaseCode)) %>%
+    group_by(MonitoringLocationIdentifier, ActivityStartDate) %>%
```

Comment:
For records with the same site-date (and more specifically, same monitoring org, characteristic name), I would expect that the majority of these seemingly-duplicated records differ in the analytical method (e.g. 1 lab sample + 1 field sample, sometimes recorded in `ResultAnalyticalMethod.MethodIdentifier` or `USGSPCode`). Here, we're not necessarily interested in those different methods and instead are focused on collapsing these records to 1 value/site-day. 

(I'm not making any suggestions, just articulating this to make sure I'm understanding the steps 🙂)

Reply:
This step is a bit wonky. When WQP reports a statcode (like mean, min, and max), then the records are returned in a long format (one row for each mean, min, max). This particular step is just to transform the relevant statcodes to our format (which is a separate column for min, mean, max).

Reply:
Oh, OK I don't think I fully appreciated that when I first looked at this PR. Makes sense, thanks!